<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HonorWallWidget</class>
 <widget class="QWidget" name="HonorWallWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>荣誉墙</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="topSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="middleLayout">
     <item>
      <widget class="QListWidget" name="honorListWidget">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>300</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>120</width>
         <height>90</height>
        </size>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightLayout">
       <item>
        <widget class="QLabel" name="previewLabel">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>220</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text">
          <string>请选择左侧的荣誉照片</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dateLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="descLabel">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="bottomSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
